applications/nrf5340_audio/include/nrf5340_audio_common.h

```diff
@@ -38,4 +38,17 @@ struct le_audio_msg {
 	enum le_audio_evt_type event;
 };
 
+enum bt_mgmt_evt_type {
+	BT_MGMT_CONNECTED,
```

Comment:
Suggest having the first one as invalid/not set/...  Makes it easier to spot errors.